src/components/Credentials/CredentialJson.js

```diff
@@ -10,7 +10,7 @@ const CredentialJson = ({ credential, textAreaRows='10' }) => {
 
 	useEffect(() => {
 		if (container) {
-			container.credentialParserRegistry.parse(credential).then((c) => {
+			container.credentialParserRegistry.parse(credential.vc || credential).then((c) => {
```

Comment:
The `credentialParserRegistry` now only has a parser for the vc+sd-jwt format. It should intelligently parse the jwt_vc_json format if applicable.